src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Subscription.java

```diff
@@ -60,4 +60,17 @@ public Boolean doesCaseHaveSubscriptions() {
         return isSmsSubscribed() || isEmailSubscribed();
     }
 
+    @JsonIgnore
+    public boolean isEmpty() {
+        return (this.wantSmsNotifications == null &&
+                this.tya == null &&
+                this.email == null &&
+                this.mobile == null &&
+                this.subscribeEmail == null &&
+                this.subscribeSms == null &&
+                this.reason == null &&
+                this.lastLoggedIntoMya == null
```

Comment:
I think only mandatory properties needs to check here. As other properties always be null

Reply:
But none of them are mandatory I don't think, unless you think otherwise?

Reply:
lastLoggedIntoMya will be null if not logged in by MYA, its optional, reason could be null all time, For some cases mobile could be null